src/formatDate.js

```diff
@@ -7,8 +7,54 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dayArr = date.split(fromFormat[3]);
+  let [year, month, day] = dayArr;
```

Comment:
Destructuring the date parts assumes a specific order which may not match the `fromFormat`. This could lead to incorrect assignments of `year`, `month`, and `day`.